modules/data-loading/partials/kafka/kafka-data-source-details.adoc

```diff
@@ -25,8 +25,9 @@ If the source cluster is configured for SSL or SASL protocols, you need to provi
 * If the source cluster uses SASL *and* SSL, you need to upload the keytab of each Kerberos principal, as well as the key store and truststore to every node of your TigerGraph cluster.
 Each file must be at the same absolute path on all nodes.
 
-The following configurations are required for admin, producer and consumer. To supply the configuration for the corresponding component, replace `<prefix>` with `source.admin`, `producer`, or `consumer`.
+The following are generic configurations required for admin, producer and consumer. To supply the configuration for the corresponding component, replace `<prefix>` with `source.admin`, `producer`, or `consumer`.
```

Comment:
Did you tried it? 
* Are `source.admin`, `producer`, or `consumer` all required? 
* Why `admin` requires `source` prefix but others are not?
* How about using `source.cluster`?

Reply:
I should have mentioned this earlier: 
* We already have a separate instructions on how to set up security (SSL), so user can just follow that instruction to set up SSL, by gadmin enabling a couple of flags, setting certificates etc.;
* This example is simply about avro/mm2, so let's try to make this example as simple as possible;
* This document was provided when SSL is not supported; now given SSL is already supported, we can just recommend user to use SSL if they are concerned about security; 
* All these settings are covered by our current regress test, e.g., cqrs_cluster 101. 
* I would recommend to remove all the wording including SASL from the document. We have a plan to support SASL, but we haven't implemented/supported it yet.

Let me know your thoughts. 
Thanks!

Reply:
> All these settings are covered by our current regress test, e.g., cqrs_cluster 101.

* cqrs_cluster 101 is for CRR with SSL, not loading from external Kafka. The SSL setting should be different because the CRR SSL setting is auto-generated.
* In the auto-generated CRR config you wrote, it is using `source.cluster.ssl` instead of `source.admin`, `producer` or `consumer`

Reply:
- No. cqrs_cluster 101 includes two test cases: test1.sh and test2.sh. test1.sh is for generic mm2; test2.sh is for crr. 
- When I designed SSL for MM2/CRR, connector_manager was used to set up ssl configuration settings to improve the usability. 
- You seemed to ask similar question when you reviewed the SSL PR, here is one of the comments:  https://github.com/tigergraph/cqrs/pull/3001/files#r1432225187 . Basically, Kafka(KafkaConn) maintains precedence of SSL settings: (source or target).cluster.ssl > admin/producer/consumer.ssl etc.. In the design, I just made it simple: all admin/producer/consumer share the same SSL settings to make things simple.

Reply:
> Basically, Kafka(KafkaConn) maintains precedence of SSL settings: (source or target).cluster.ssl > admin/producer/consumer.ssl etc.

* Have we explained this to the user? That's why I'm asking to improve the doc.
* Even if we use `admin`, `consumer` and `producer` separately, there's still some mistakes in the doc: why `source.admin` but not no `source.` for `consumer` and `producer`?